docs/best-practices/data-import.rst

```diff
@@ -66,23 +70,24 @@ configuration will be used:
 :replicas:
   1
 
-We recommend you choose the number of shards wisely. They depend on the
-number of nodes in the cluster as well as on the amount of data that goes into
-the table.
+We recommend you choose the number of shards wisely. It depends on the number
+of nodes in the cluster as well as on the amount of data that goes into the
+table.
 
 .. CAUTION::
 
   Be careful, you cannot change the number of shards once the table is created!
 
-Assuming there are 6 nodes in the cluster, and we put 2 shards on each node
-giving us a total of 12 shards, which should be enough for millions of records.
+Assuming there are 6 nodes in the cluster, we put 2 shards on each node,
+giving us a total of 12 shards. This should be enough for millions of
```

Comment:
"us" makes me wonder if it's a big issue to also correct the use of pronouns in this doc to match https://developers.google.com/style/pronouns (no worries if not!)

Reply:
Yup! I think there were only a few 'wes'